profiles/Octaviya-setup.yaml

```diff
@@ -0,0 +1,280 @@
+# https://elanthipedia.play.net/Cleric#Circle_Requirements
+# 3 Magics: Arcana, Augmentation, Attunement
+# 3 Weapons (only 1 is needed to circle): Brawling, Heavy Thrown, Large Blunt
+# 1 Armor: Chain Armor (does not include Shield Usage)
+# 4 Survivals: Athletics, Perception, Evasion, Outdoorsmanship
+# 4 Lores (only 3 are needed to circle): Appraisal, Tactics, Scholarship, Mechanical Lore
+
+# Eventually you will need to train 1 more Weapon to circle (for a total of 2)
+
+# Eventually you will need to train 2 more Magics to circle (for a total of 5)
+# However, you will probably end up training them all since they each have very useful spells
+# The other Magic skills are: Targeted Magic, Warding, Debilitation, Utility
+
+# Eventually you will need to train 1 more Lore to circle (for a total of 4)
+
+
+hunting_file_list:
+- setup
+
+# Hunting settings
+training_manager_hunting_priority: true
+training_manager_priority_skills:
+- Brawling
+- Light Thrown
+- Small Edged
+hunting_info:
+- :zone: rats
+  # :duration: 30
+  stop_on:
+  - Brawling
+  - Light Thrown
+  - Small Edged
+prehunt_buffs: 7958
```

Comment:
I don't think you want to buff here if you're hunting rats. I think that room can have leucros in it. Regardless, it's far away from the rats zone.